doc/topics/cloud/config.rst

```diff
@@ -27,10 +27,12 @@ cloud is operating on.
 Minion Configuration
 ====================
 
-The default minion configuration is set up in this file. Minions created by
-salt-cloud derive their configuration from this file.  Almost all parameters
-found in :ref:`Configuring the Salt Minion <configuration-salt-minion>` can
-be used here.
+The default minion configuration is set up in this file or alternatively used in
+any ``.conf`` file listed in the ``/etc/salt/cloud.maps.d/``
+:ref:`map file <salt-cloud-map>` directory. Minions created by salt-cloud
```

Comment:
I wonder if it would be better to simply add the map file to this portion of the docs: 

> Similar to most other settings, Minion configuration settings are inherited across configuration files. For example, the master setting might be contained in the main cloud configuration file as demonstrated above, but additional settings can be placed in the provider or profile:

So instead of just provider or profile we include all three > provider, profile, and map file. What you think?

If you only call map files out right there it doesn't account for profile and provider files where you can also set the master config, and in the paragraph afterwards we mention that you can specify this setting in other places, just hvaen't clarified the map file is included in this.

Reply:
@Ch3LL I agree your option will work fine and I can make that change if you'd like.

Reply:
yeah that would be great. thanks